### ISSUE TITLE
Cannot update repo

### ISSUE THREAD

**Issue by lcolladotor** [dzhang32/dasper#3]
Hi David,

I tried fixing https://github.com/lcolladotor/derfinder/pull/42 and updating the GHA workflow to use biocthis 1.3.6 but I ran into:

```bash
$ git remote -v
origin	git@github.com:dzhang32/dasper.git (fetch)
origin	git@github.com:dzhang32/dasper.git (push)
13:15 dasper $ git push origin
ERROR: Permission to dzhang32/dasper.git denied to lcolladotor.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Best,
Leo

**Comment by lcolladotor** [comment]
I can send a PR if you prefer

**Comment by dzhang32** [comment]
Hi Leo, 

Thanks a lot for helping with this. 

I have invited you as a collaborator on the repo, which hopefully allows you to push this change. 

Let me know if that works. 

Best, 

David

**Comment by lcolladotor** [comment]
Thanks! I still don't have access from to write at git@git.bioconductor.org:packages/dasper.git, but well, that's ok. Can you pull https://github.com/dzhang32/dasper/commit/c02353683988c9e0e9f7e4be0f42031cbe0037cc and push it to BioC? Thanks!

By the way, why do you have `pkgdown = false` on the GHA workflow?

**Comment by dzhang32** [comment]
Thanks and pushed. 

Re. `pkgdown`, I came across an environment error wherein running `pkgdown::deploy_to_branch(new_process = FALSE)` failed in the GHA runner, but was successful when executed locally. I think it had something to do with the incorporation of `basilisk` but never did manage to resolve it. I opted to just run `pkgdown` locally after each update and will come back to it at some point.
